tests/server/googleLogin.test.js

```diff
@@ -0,0 +1,89 @@
+import { insertUser } from "../../server/api/googleLogin";
```

Comment:
Coverage is currently ~61%

I'd try to get it up if you can. Take a look at the server tests on branch for wpp-73 to see how some of it can be handled

Worst case scenario, I can write a few more tests for it in that user story